packages/i18n/src/locales/en/options/holmes.yml

```diff
@@ -7,7 +7,7 @@ lengthRatio:
   description: Controls the length of the crown and ear flaps
 gores:
   title: Number of gores
-  description: The number of gores used to construct the semi-sphere
+  description: The number of gores used to construct the crown
```

Comment:
Never heard of a "gore" before!

This change makes perfect sense to me and also seems in line with [the documentation page](https://freesewing.org/docs/patterns/holmes/options/gores/) for this option:
> Controls the number of panels there are in the crown

Reply:
Gore in this case is just triangle-shaped. It's not really the right term in the context of a hat and sewing but it makes sense for the code etc,